protocol/query/method/message/data/dataElectionSetup.json

```diff
@@ -0,0 +1,87 @@
+{
+	"$schema": "http://json-schema.org/draft-07/schema#",
+	"$id": "https://raw.githubusercontent.com/dedis/student_21_pop/master/protocol/query/method/message/data/dataSetupElection.json",
+	"description": "Match an ElectionSetup query",
+	"type": "object",
+
+	"properties": {
+		"object": { "const": "election" },
+		"action": { "const": "setup" },
+		"id": {
+			"type": "string",
+			"contentEncoding": "base64",
+			"$comment": "Hash : SHA256('Election'||lao_id||created_at||name)"
+		},
+		"lao": { 
+			"type": "string",
+			"contentEncoding": "base64",
+			"$comment": "ID of the LAO"
+			},
+		"name": { 
+			"type": "string",
+			"$comment": "name of the election"
+		},
+		"version": { 
+			"type": "string",
+			"$comment": "features/implementation identifier"
+		},
+		"created_at": {
+			"description": "[Timestamp] time created in UTC",
+			"type": "integer",
+			"minimum": 0
+		},
+		"start_time": {
+			"description": "[Timestamp] start time of the election in UTC",
+			"type": "integer",
+			"minimum": 0
+		},
+		"end_time": {
+			"description": "[Timestamp] end time of the election in UTC",
+			"type": "integer",
+			"minimum": 0
+		},
+		"questions": {
+			"description": "Array of questions",
+			"type": "array",
+			"items": {
+				"type": "object",
+				"properties": {
+					"id": {
+						"type": "string",
+						"contentEncoding": "base64",
+						"$comment": "ID of the question"
+					},
+					"question": {
+						"type": "string",
+						"$comment": "voting topic"
+					},
+					"voting_method": {
+						"type": "string",
+						"enum": ["Plurality", "Approval"],
+						"$comment": "supported voting method"
+					},
+					"ballot_options": {
+						"description": "[Array[String]] ballot options",
+						"type": "array",
+						"items": {
+							"type": "string",
+							"$comment": "ballot option"
+						},
+						"minItems": 1,
```

Comment:
Perhaps minItems should be set to 2? Since a question with only 1 possible answer is not really useful, but maybe this doesn't have to be enforced at the protocol level

Reply:
I agree, it makes more sense. I updated it!